src/libs/zuora.ts

```diff
@@ -133,10 +133,19 @@ Additional Comms                                                   # reserved fo
     FROM
       RatePlanCharge
     WHERE
-      Product.ProductType__c = 'Newspaper - National Delivery' 
-      and RatePlanCharge.name = '${dayOfTheWeekName}' 
-      and RatePlanCharge.effectiveStartDate <= '${date}'
-      and (RatePlanCharge.effectiveEndDate >= '${date}' or (Subscription.autoRenew = true and Subscription.status = 'Active'))
+      (Subscription.Status = 'Active' OR Subscription.Status = 'Cancelled') AND
```

Comment:
bascially want to ignore expired i.e. superseded subscription versions.
Cancelled just means there can be no further amendments and auto renewal won't happen etc, but it doesn't mean the customer isn't still serving a cancellation period.
If a reader requests cancellation it's often for the end of the term or period they've already paid for.